WcaOnRails/app/assets/stylesheets/delegates.scss

```diff
@@ -1,7 +1,8 @@
 tr.senior-delegate {
-  background-color: black;
-  color: white;
-  font-weight: bold;
+  background-color: #e9e9e9;
+  color: black;
+  //font-weight: bold;
```

Comment:
why did you comment this out instead of removing it?

Reply:
Sorry, bad habit 🤣, Ill fix that now